src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,23 @@
+@import "../utils/variables";
+
+html {
+  font-family: Roboto, sans-serif;
+  color: $main-text-color;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
box-sizing shouldn't be with reset styles, add box-sizing to elements where you need it

Reply:
I think normalize file, is the place for it